noxfile.py

```diff
@@ -53,28 +59,41 @@ def doc(session):
     session.install("-e", ".[doc]")
     session.cd("doc")
     session.run("sphinx-build", "-M", target, ".", "_build", *options)
+    session.run("open", "_build/html/index.html")
```

Comment:
Please, let's not do this by default. I would not mind an option.

Reply:
I added this and other options